CMakeLists.txt

```diff
@@ -265,74 +254,76 @@ if (DFTRACER_BUILD_PYTHON_BINDINGS)
         #add_subdirectory(dependency/py11bind)
 endif()
 
+find_package(MPI COMPONENTS CXX QUIET)
+if (MPI_CXX_FOUND AND DFTRACER_ENABLE_MPI)
```

Comment:
Can you add a high level knob to enable detection based on user's request? The knob would be disabled by default

Reply:
DONE